RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -24,7 +26,9 @@
     "drop recoPFClusters_*_*_*",
     "keep recoElectronSeeds_*_*_*",
     "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
+    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
+    'keep recoPhotons_gedPhotonsTmp_*_*',
```

Comment:
I should have said "going to be added".
Is it intentional for the HI setup to use the *Tmp names? Unless there is a near term plan to add relinking, the current naming setup looks awkward.